app/manage.py

```diff
@@ -18,12 +18,8 @@ def run(self):
 
             print("~~~~~~~~~~ Searching for Master Address ~~~~~~~~~~")
 
-            if app.config['IS_USING_BITCOIN']:
-                master_address_type = "BITCOIN_MASTER"
-                master_address = app.config['BITCOIN_MASTER_WALLET_ADDRESS']
-            else:
-                master_address_type = "MASTER"
-                master_address = app.config['MASTER_WALLET_ADDRESS']
```

Comment:
@enjeyw Have we removed support for Bitcoin?

Reply:
not properly yet, but I think we should deprecate - it doesn't fit our current strategic plans?